<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="drop_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>991</width>
      <height>651</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_detection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>981</width>
       <height>631</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#141414;
border-top-left-radius:50px;
border-bottom-right-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_detection_button">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>590</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 8px;
background:#D0F266;
height: 30px;
color: #000;

}

QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>Stop Detection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="min_butto">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>10</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{

border-radius: 13.5px;
background: #141414;
color: #ACACAC;
border: 2px solid #242424;

}

QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cls_button">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>10</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{

border-radius: 13.5px;
background: #141414;
color: #D2F565;
border: 2px solid #242424;

}
QPushButton:hover{
background-color:#e63946;
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
